### PR TITLE
fix(onboard/contributing): report updates of the contribution made on…

### PR DESCRIPTION
**Pull request by tiloyi**
… a previous branch

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

This PR contains the transfer of changes made on the Contributing pages to a previous branch (css_feat-convention)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: src/pages/Contributing/Install/index.md [hunk 1]
```diff
@@ -30,25 +30,25 @@ please [read the docs](https://github.com/lerna/lerna/) to learn more about it.
 
 ### Clone the repo and start the development environement
 
-clone the repo :
```
Comment (gael-boyenval) [inline src/pages/Contributing/Install/index.md]: don't remove two spaces they are line breaks in markdown

File: src/pages/Contributing/Install/index.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ please folow the this [guide](https://nodejs.org/en/download/package-manager/)
 
 ### Note about the project architecture
 
-The project Use a monorepo architecture using [lerna](https://github.com/lerna/lerna/).  
-Lerna help us to manage the distribution and the versionning of multiple packages into the same repository.
```
Comment (gael-boyenval) [inline src/pages/Contributing/Install/index.md]: (everywhere) don't remove two spaces they are line breaks in markdown

File: src/pages/Contributing/Install/index.md (hunk 1 above)
Comment (tiloyi) [reply src/pages/Contributing/Install/index.md]: Fixed

File: src/pages/Contributing/Install/index.md (hunk 2 above)
Comment (tiloyi) [reply src/pages/Contributing/Install/index.md]: Fixed

**Pull request merged**